sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/models/__init__.py

```diff
@@ -6,85 +6,117 @@
+from ._models_py3 import BasePrediction
+from ._models_py3 import BaseResolution
+from ._models_py3 import BooleanResolution
+from ._models_py3 import ConversationAnalysisOptions
+from ._models_py3 import ConversationCallingOptions
+from ._models_py3 import ConversationEntity
+from ._models_py3 import ConversationIntent
+from ._models_py3 import ConversationItemBase
+from ._models_py3 import ConversationParameters
+from ._models_py3 import ConversationPrediction
+from ._models_py3 import ConversationResult
+from ._models_py3 import ConversationTargetIntentResult
+from ._models_py3 import CurrencyResolution
+from ._models_py3 import CustomConversationTaskParameters
+from ._models_py3 import CustomConversationalTask
+from ._models_py3 import CustomConversationalTaskResult
+from ._models_py3 import DateTimeResolution
+from ._models_py3 import EntitySubtype
+from ._models_py3 import Error
+from ._models_py3 import ErrorResponse
+from ._models_py3 import InformationResolution
+from ._models_py3 import InnerErrorModel
+from ._models_py3 import KnowledgeBaseAnswer
+from ._models_py3 import KnowledgeBaseAnswerDialog
+from ._models_py3 import KnowledgeBaseAnswerPrompt
+from ._models_py3 import KnowledgeBaseAnswers
+from ._models_py3 import LUISCallingOptions
+from ._models_py3 import LUISParameters
+from ._models_py3 import LUISTargetIntentResult
+from ._models_py3 import LengthResolution
+from ._models_py3 import ListKey
+from ._models_py3 import NoneLinkedTargetIntentResult
+from ._models_py3 import NumberResolution
+from ._models_py3 import NumericRangeResolution
+from ._models_py3 import OrchestratorPrediction
+from ._models_py3 import OrdinalResolution
+from ._models_py3 import QuantityResolution
+from ._models_py3 import QuestionAnsweringParameters
+from ._models_py3 import QuestionAnsweringTargetIntentResult
+from ._models_py3 import SpeedResolution
+from ._models_py3 import TargetIntentResult
+from ._models_py3 import TemperatureResolution
+from ._models_py3 import TemporalSpanResolution
+from ._models_py3 import TextConversationItem
+from ._models_py3 import VolumeResolution
+from ._models_py3 import WeightResolution
+
 
 from ._conversation_analysis_client_enums import (
+    AgeUnit,
+    AnalyzeConversationTaskKind,
+    AnalyzeConversationTaskResultsKind,
+    AreaUnit,
+    DateTimeSubKind,
     ErrorCode,
+    ExtraInformationKind,
+    InformationUnit,
     InnerErrorCode,
+    LengthUnit,
+    Modality,
+    Modifier,
+    NumberKind,
     ProjectKind,
+    RangeKind,
+    RelativeTo,
+    ResolutionKind,
+    SpeedUnit,
     TargetKind,
+    TemperatureUnit,
+    VolumeUnit,
+    WeightUnit,
 )
 
 __all__ = [
+    'AgeResolution',
     'AnalysisParameters',
     'AnalyzeConversationResult',
+    'AnalyzeConversationTask',
+    'AnalyzeConversationTaskResult',
     'AnswerSpan',
+    'AreaResolution',
+    'BaseExtraInformation',
     'BasePrediction',
+    'BaseResolution',
+    'BooleanResolution',
     'ConversationAnalysisOptions',
```

Comment:
What's the difference between ConversationAnalysisOptions and AnalyzeConversationTask?